src/WCSimDetectorMessenger.cc

```diff
@@ -399,7 +419,6 @@ WCSimDetectorMessenger::WCSimDetectorMessenger(WCSimDetectorConstruction* WCSimD
 				    "BoxandLine12inchHQE "
 				    "PMT3inchR12199_02 "
 				    "PMT3inchR14374 "
-            "PMT3inchR14374_WCTE "
```

Comment:
Please add these `PMT3inchR14374_WCTE` lines back in - WCTE geometries can't be run without them, as [seen in the CI](https://github.com/WCSim/WCSim/actions/runs/11232171878/job/31223532823?pr=468#step:12:206)

Reply:
done